pkg/jobrunaggregator/jobruntablecreator/table_creator.go

```diff
@@ -0,0 +1,37 @@
+package jobruntablecreator
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"cloud.google.com/go/bigquery"
+
+	"github.com/openshift/ci-tools/pkg/jobrunaggregator/jobrunaggregatorapi"
+	"github.com/openshift/ci-tools/pkg/jobrunaggregator/jobrunaggregatorlib"
+)
+
+type allJobRunTableCreatorOptions struct {
+	ciDataClient jobrunaggregatorlib.CIDataClient
+	ciDataSet    *bigquery.Dataset
+}
+
+func (r *allJobRunTableCreatorOptions) Run(ctx context.Context) error {
+
+	// Create JobRunTable
+	jobRunTable := r.ciDataSet.Table(jobrunaggregatorlib.JobsTableName)
+	_, err := jobRunTable.Metadata(ctx)
+	if err != nil {
+		schema, err := bigquery.InferSchema(jobrunaggregatorapi.JobRow{})
```

Comment:
Have you been able to successfully run this on your own data set, and did the schema compare to David's actual table properly?

Reply:
yes.  I also added commands I used in the README.md.  Also, added a comment about inferring the schema (without field comments) where the schema/struct is defined.